CONTRIBUTING.rst

```diff
@@ -35,9 +41,9 @@ Checklist
   already, add yourself to the AUTHORS file following the convention found
   therein.
 * We have 100% test coverage - include tests to maintain this!
-* Before submitting code ensure coding standards and test coverage by running::
+* **Before submitting code ensure coding standards and test coverage by running**::
 
     $ make check
 
 * If in doubt, ask a question. The only stupid question is the one that's never asked.
-* Have fun!
```

Comment:
Maybe add "We hope that when people ask questions you can answer you will hel them out by doing so"?